_posts/2018-04-11-making-a-cockpit-application.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to turn your Cockpit page into an Application
```

Comment:
I know what that means, but maybe point out the intended effect a bit more? Proposal: "Discover your Cockpit extension on the Applications page". I'm fine with the title as-is, though.